R/dominantTaxa.R

```diff
@@ -0,0 +1,210 @@
+#' Dominant Taxa
+#'
+#' This function returns the most dominant taxa.
+#'
+#' @param x
+#' A \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#' object
+#'
+#' @param abund_values A single character value for selecting the
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assay}}
+#'   to use for prevalence calculation.
+#'
+#' @param rank A single character defining a taxonomic rank. Must be a value
+#' of the output of \code{taxonomicRanks()}.
+#'
+#' @param group With group, it is possible to group the observations. Must
+#' be a one of the column names of colData.
+#'
+#' @param name A name for the column of the colData where the dominant taxa
+#' will be stored in. Must be a string.
+#'
+#' @details
+#' \code{dominantTaxa} extracts the most abundant taxa
+#' in a \code{\link[=SummarizedExperiment-class]{SummarizedExperiment}} object.
+#'
+#' @return \code{x} with additional \code{\link{colData}} named
```

Comment:
Two different return types now

Reply:
Changed it to 

#' @return \code{dominantTaxa} returns \code{x} with additional \code{\link{colData}} named
#'   \code{*name*}. \code{getDominantTaxa} returns a tibble containing dominant taxa in
#'   a column named \code{*name*} and its abundance in the data set.